### PR TITLE
ocproxy: init at 1.50

### PR DESCRIPTION
**Pull request by jokogr**
###### Motivation for this change

This package can be combined with `openconnect` and act as a proxy, so that only specific network activity is forwarded to openconnect.
###### Things done
- [x] Tested using sandboxing
  ([nix.useChroot](http://nixos.org/nixos/manual/options.html#opt-nix.useChroot) on NixOS,
    or option `build-use-chroot` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
  - [x] NixOS
  - [ ] OS X
  - [ ] Linux
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/tools/networking/ocproxy/default.nix [hunk 1]
```diff
@@ -0,0 +1,34 @@
+{ stdenv, fetchFromGitHub, autoconf, automake, libevent }:
+
+let
+  version = "1.50";
```
Comment (zimbatm) [inline pkgs/tools/networking/ocproxy/default.nix]: nitpick: it's quite common to put the version in the derivation directly below the name.

**Comment by zimbatm** [comment]
Looking good

File: pkgs/tools/networking/ocproxy/default.nix (hunk 1 above)
Comment (jokogr) [inline pkgs/tools/networking/ocproxy/default.nix]: Hope you meant above, fixed it at d861911

Comment (zimbatm) [inline pkgs/tools/networking/ocproxy/default.nix]: Close enough :)

**Pull request merged**